server/game/effects.js

```diff
@@ -59,12 +59,14 @@ const Effects = {
     alert: () => EffectBuilder.card.static('alert'),
     bound: () => EffectBuilder.card.static('bound'),
     canGuard: () => EffectBuilder.card.static('canGuard'),
-    concealed: () => EffectBuilder.card.static('concealed'),
+    cannotBeSpellTarget: () => EffectBuilder.card.static('cannotBeSpellTarget'),
```

Comment:
separate the different aspects of concealed out so they can be used independently. concealed just applies all of them together